src/Metadata/AbstractPropertyMetadata.php

```diff
@@ -108,7 +108,7 @@ public function hasCustomInformation(string $key): bool
      *
      * @return mixed The information in whatever format it has been set
      */
-    public function getCustomInformation(string $key)
+    public function getCustomInformation(string $key): mixed
```

Comment:
sorry for the back and forth. i realized we are still in pre - 1.0 versions so the BC break here seems acceptable to me. its better than disabling a general cs fixer rule imho.